README.md

```diff
@@ -128,10 +125,10 @@ but the JSON you receive from the server looks like this:
 }
 ```
 
-you would want that JSON parsing to fail and return `.None` instead of a
-`User` object. That base concept means that when you're dealing with an actual
-`User` object, you can be sure that when you ask for `user.id`, you're going
-to get the type you expect.
+you would want that JSON parsing to fail and return some error state instead of
+a `User` object. That base concept means that when you're dealing with an
```

Comment:
In Argo, you'll either get a full and valid `User`, or a reason why the User couldn't be constructed.